django_session_jwt/test.py

```diff
@@ -1,5 +1,7 @@
 from importlib import import_module
 
+from jwt.exceptions import DecodeError
```

Comment:
This doesn't appear to be used

Reply:
You're right. I removed the exception handling but not the import. Gone.